tasks/opsep/update-l2pao-to-aliased-l1pao/SignFromJson.s.sol

```diff
@@ -0,0 +1,188 @@
+    // Aliased L1PAO
+    address constant aliasedL1PAO = 0x2FC3ffc903729a0f03966b917003800B145F67F3;
+    address constant unaliasedL1PAO = 0x1Eb2fFc903729a0F03966B917003800b145F56E2;
+    uint160 private constant ALIASING_OFFSET = uint160(0x1111000000000000000000000000000000001111);
+
+    // Known EOAs to exclude from safety checks.
+    address constant l2OutputOracleProposer = 0x49277EE36A024120Ee218127354c4a3591dc90A9; // cast call $L2OO "PROPOSER()(address)"
+    address constant l2OutputOracleChallenger = 0xfd1D2e729aE8eEe2E146c033bf4400fE75284301; // In registry addresses.
+    address constant systemConfigOwner = 0xfd1D2e729aE8eEe2E146c033bf4400fE75284301; // In registry addresses.
+    address constant batchSenderAddress = 0x8F23BB38F531600e5d8FDDaAEC41F13FaB46E98c; // In registry genesis-system-configs
+    address constant p2pSequencerAddress = 0x57CACBB0d30b01eb2462e5dC940c161aff3230D3; // cast call $SystemConfig "unsafeBlockSigner()(address)"
+    address constant batchInboxAddress = 0xff00000000000000000000000000000011155420; // In registry yaml.
+    
+    Types.ContractSet proxies;
+
+    // This gives the initial fork, so we can use it to switch back after fetching data.
+    uint256 initialFork;
+
+    /// @notice Sets up the contract
+    function setUp() public {
+        proxies = _getContractSet();
+
+        // Fetch variables that are not expected to change from an older block.
+        initialFork = vm.activeFork();
+        vm.createSelectFork(vm.envString("ETH_RPC_URL"), 5705332); // This block is from April 15 2024 at 11:40am PT.
+
+        require(unaliasedL1PAO == undoL1ToL2Alias(aliasedL1PAO), "setUp-100");
+        require(aliasedL1PAO == applyL1ToL2Alias(unaliasedL1PAO), "setUp-101");
+
+        vm.selectFork(initialFork);
+    }
+
+    function checkL2PAOSafe() internal view {
+        console.log("Running assertions on the L2PAO Safe");
+
+        address[] memory l2paoSafeOwners = l2paoSafe.getOwners();
+        require(l2paoSafeOwners.length == 1, "checkL2PAOSafe-100");
+        require(l2paoSafeOwners[0] == l2PaoSafeEOA, "checkL2PAOSafe-101"); // stays the same
+        require(l2paoSafe.getThreshold() == 1, "checkL2PAOSafe-102");
+    }
+
+    function checkL2PA() internal view {
+        console.log("Running assertions on the L2PA");
+
+        address l2paOwner = l2pa.owner();
+        require(l2paOwner == aliasedL1PAO, "checkL2PA-200");
+        require(unaliasedL1PAO == undoL1ToL2Alias(l2paOwner), "checkL2PA-201");
+    }
+
+    function applyL1ToL2Alias(address l1Address) public pure returns (address l2Address) {
+        unchecked {
+            l2Address = address(uint160(l1Address) + ALIASING_OFFSET);
+        }
+    }
+
+    function undoL1ToL2Alias(address l2Address) public pure returns (address l1Address) {
+        unchecked {
+            l1Address = address(uint160(l2Address) - ALIASING_OFFSET);
+        }
+    }
```

Comment:
As mentioned above, we can delete this in favor of importing the `AddressAliasHelper` library

Reply:
done: https://github.com/ethereum-optimism/superchain-ops/pull/204/commits/5055a9049e26cf7cd4c847be600d41042295e377